tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java

```diff
@@ -105,13 +106,13 @@ public void singleMasterJournalCrashIntegration() throws Exception {
   @Test
   public void multiMasterJournalStopIntegration() throws Exception {
     MultiMasterLocalAlluxioCluster cluster = setupMultiMasterCluster();
+    CommonUtils.sleepMs(TEST_TIME_MS);
```

Comment:
This waiting is not for the cluster to start up, but to actually let the background thread to run a bit. I added a comment.